### ISSUE TITLE
Feature request - tooltips when hovering over item in main view

### ISSUE THREAD

**Issue by mdear-lsn** [newlooper/VirtualSpace#69]
When a desktop on the main view may have many (ie. hundreds) of windows, categorizing them is very tedious as I have to follow the "create rule" flow just to see what the window actually is.

Please have a tooltip appear on brief hover that at least quotes window title / app.

**Comment by micoro** [comment]
Workaround suggestion: Autohotkey has a utility (windowspy.ahk) to do exactly this. As you move the mouse on different windows it tells you informations on that window.

[Window Spy in AHK docs](https://amourspirit.github.io/AutoHotkey-Snippit/WindowSpy.html)
[Autohotkey site and download](https://www.autohotkey.com/)

**Comment by mdear-lsn** [comment]
As I continue testing VirtualSpace, I am liking it more and more.  Despite lacking this feature, the inbuild Windows Task View has this feature and it's pretty convenient to go between a Task View and VirtualSpace windows.  I find VirtualSpace dovetails rather nicely with TaskView and reinforces it.  Great work !

**Comment by mdear-lsn** [comment]
> Workaround suggestion: Autohotkey has a utility (windowspy.ahk) to do exactly this. As you move the mouse on different windows it tells you informations on that window.
> 
> [Window Spy in AHK docs](https://amourspirit.github.io/AutoHotkey-Snippit/WindowSpy.html) [Autohotkey site and download](https://www.autohotkey.com/)

I took a quick look at it doesn't appear this AHK script would run on minimized windows in the VS main view, ideally I'd like to see the window title as a tooltip that would appear after a hover of around a second or so.

**Comment by mdear-lsn** [comment]
I tried to hover over various windows on the VS main view, I only found AHK Window Spy showed two different titles : one for the grey space between the workspaces (background) and another when the cursor was inside the miniaturized desktop thumbnails.

What I''m looking for is a tooltip that doesn't necessarily show the workspace name (which is already printed and readable on each workspace) but rather the title of each individual miniaturized window thumbnail (which autoscales down as more windows are added to the workspace, making them harder to recognize) in the main VS window.    

I know that the window titles are shown in the inbuilt Task View display, but it's not convenient to view them there because there is no way to autocreate a rule from a window.

This feature would make an excellent product even better and more efficient to use for those of us who wear multiple hats and have powerful machines to hold open many windows.  Being able to quickly switch contexts to different workspaces is essential for our productivity.

![2023-04-19 22 44 54](https://user-images.githubusercontent.com/124737904/233476634-84ac0fc7-0abf-4d0c-a8eb-615a9f11d4d1.jpg)
![2023-04-19 22 45 03](https://user-images.githubusercontent.com/124737904/233476692-ead4a0b0-91b8-4e54-ba80-165a452021ab.jpg)
